legislators-social-media.yaml

```diff
@@ -3995,6 +4000,7 @@
     thomas: '01925'
     govtrack: 412269
   social:
+    twitter: BillCassidy
```

Comment:
This is a campaign account and links to billcassidy.com from their bio. I don't see an official account linked from https://www.cassidy.senate.gov. Please take this one out of the PR.

Reply:
Sure thing. I've now removed it.